cms/static/js/views/experiment_group_edit.js

```diff
@@ -13,8 +13,8 @@ function(BaseView, _, str, gettext) {
         events: {
             'click .action-close': 'removeGroup',
             'change .group-name': 'changeName',
-            'focus .groups-fields input': 'onFocus',
-            'blur .groups-fields input': 'onBlur'
+            'focus .group-name': 'onFocus',
```

Comment:
Note to reviewers: the element with class group-fields is outside the template used for this view.

The input has class "group-name", so switching to that to be consistent with tests and the "changeName" event handler above.